src/fragments/start/getting-started/reactnative/integrate.mdx

```diff
@@ -0,0 +1,411 @@
+In this section you’ll integrate Amplify DataStore with your app, and learn to use the generated data model to create, update, query, and delete Todo items by building an app. You can find the source code of the final Todo App [on our GitHub repository](https://github.com/chintannp/amplified_todo_rn).
+
+## Boilerplate UI
+First, we will create some files and place the boilerplate UI code in it. You might find your IDE complains about unreferenced declarations but, don’t worry, we’ll get around to fixing those as we fill out our app with more functionality.
+
+#### App.js
+First, replace the contents of your `App.js` file with the following UI boilerplate code. Here, we are importing our Home component. Additionally, we are configuring Amplify using `Amplify.configure`. You might have already done this in previous step while setting up the project.
+
+```js
+import React from 'react';
+import { StatusBar, StyleSheet, View } from 'react-native';
+import Amplify from 'aws-amplify';
+import Home from './src/Home';
+
+import awsconfig from './src/aws-exports';
+
+Amplify.configure(awsconfig);
+
+export default function App() {
+  return (
+    <View style={styles.container}>
+      <StatusBar />
+      <Home />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#fff',
+    flex: 1,
+  },
+});
+```
+
+#### Home.js
+Next, we will create a Home component for our app which will implement most of the CRUD functionality for our app. Create a new file `Home.js` at location `src/Home.js`. Here, we have total four components defined :
+
+1. **Header**: Simple Header component with Title.
+2. **AddModal**: Used for displaying a Modal when a new Todo needs to be added.
+3. **TodoList**: Used for displaying the list of Todos.
+4. **Home**: Default component that wraps all the above component and a Button for adding a new Todo item.
+
+Copy the below boilerplate UI code into the newly create file.
+
+```js
+import React, { useState, useEffect } from 'react';
+import {
+  FlatList,
+  Modal,
+  Pressable,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+import { DataStore } from 'aws-amplify';
+import { Todo } from './models';
+
+const Header = () => (
+  <View style={styles.headerContainer}>
+    <Text style={styles.headerTitle}>My Todo List</Text>
+  </View>
+);
+
+const AddTodoModal = ({ modalVisible, setModalVisible }) => {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+
+  async function addTodo() {
+  }
```

Comment:
Nit: I would suggest leaving some comments in these empty functions indicating that they are intentionally empty for now as code will be added in a later step

Reply:
Added the comment "to be filled in a later step". Thanks Chris